File: Kritika Tripathi/Basics Of Programming/Function and Arrays/barchart.cpp
#include<bits/stdc++.h>
using namespace std;


int main(){
    //write your code here
    int n;
    cin>>n;

    int arr[n];

    for(int i=0; i<n; i++){
        cin>>arr[i];
    }
    int maxn = INT_MIN;
    for(int i=0; i<n; i++){
        maxn = max(arr[i], maxn);
    }

    int i=0, count = maxn; // 6
    for(int i=0; i<maxn; i++, count--){  // 0 7 //1 6 //2 5 //3 4 //2 3 //1 2
        for(int j=0; j<n; j++){ //0 to 4
            if(arr[j]==count){ 
                cout<<"*\t";
                arr[j]--;
            }
            else{
                cout<<"\t";
            }
        }
        cout<<endl;
    }
    
}

File: V.Vennela/Basics of Programming/Functions and Arrays/BarChart.java
import java.io.*;
import java.util.*;

public class BarChart{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int a[] = new int[n];
    for(int i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
    }
    int max=a[0];
    for(int i=1;i<n;i++)
    {
        if(a[i]>max)
            max=a[i];
    }
    int x=max;
    for(int i=0;i<x;i++)
    {
        for(int j=0;j<n;j++)
        {
            if(a[j]==max)
            {
                System.out.print("*\t");
                a[j]=a[j]-1;
            }
            else
            {
                 System.out.print("\t");
            }
             
        }
        max--;  
        System.out.println();
    }
 }

}
